src/debugAdapter/goDebug.ts

```diff
@@ -1596,30 +1597,42 @@ export class GoDebugSession extends LoggingDebugSession {
 
 	protected evaluateRequest(response: DebugProtocol.EvaluateResponse, args: DebugProtocol.EvaluateArguments): void {
 		log('EvaluateRequest');
-		this.evaluateRequestImpl(args).then(
-			(out) => {
-				const variable = this.delve.isApiV1 ? <DebugVariable>out : (<EvalOut>out).Variable;
-				// #2326: Set the fully qualified name for variable mapping
-				variable.fullyQualifiedName = variable.name;
-				response.body = this.convertDebugVariableToProtocolVariable(variable);
+		const re = new RegExp(/\w+(?=\(.*\))/, 'g');
```

Comment:
Please add a detailed comment here (or above the `evaluateRequest` method) explaining this contract: what exactly we're looking for in the eval request to interpret as a 'call', and how it's treated. A reader writing a test for this should be able to read the comment and understand how to craft test DAP messages that would trigger this behavior, what its limitations and corner cases are, etc.